elixir/apps/domain/test/domain/flows_test.exs

```diff
@@ -173,8 +173,7 @@ defmodule Domain.FlowsTest do
       Fixtures.Actors.create_membership(account: account, actor: actor, group: actor_group2)
 
       time = Time.utc_now()
-      one_hour_ago = Time.add(time, -1, :hour)
-      one_hour_in_future = Time.add(time, 1, :hour)
+      midnight = Time.from_iso8601!("23:59:59.999999")
 
       date = Date.utc_today()
       day_of_week = Enum.at(~w[M T W R F S U], Date.day_of_week(date) - 1)
```

Comment:
This still can be flaky at the midnight, to fix this we can just add two days to the range:
```suggestion
      day_of_week_today = Enum.at(~w[M T W R F S U], Date.day_of_week(date) - 1)
      day_of_week_tomorrow = Enum.at(~w[M T W R F S U], Date.day_of_week(date))
```